OKR/GO_CORE.md

```diff
@@ -2,6 +2,29 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2019 Q1
+
+### Data transfer is fast and efficient
+* It takes less than 48 hours to transfer 1TB dataset over Fast Ethernet (100Mbps)
+* Bitswap refactor + improvements reduce number of duplicate blocks downloaded by 75%
+ 
+### Adding and providing large data sets works efficiently
+* Create Provider Strategies system that is solely responsible for providing
+* Implement provider strategy such that a user can add (and provide) npm or tr-wikipedia without turning off providing and without significantly impacting finding content 
+
+### Datastore works efficiently for large data sets
+* Final go/no-go decision on Badger (full design comparison vs other options + final decision)
+ 
+### go-ipfs becomes a well maintained project
+* Every non-trivial PR is first reviewed by someone other than @Stebalien.
+* A work-tracking process is adopted that allows the team to easily track the highest priority unclaimed issues, see the status of in progress work, and know which PRs need review, easily. 
+ 
+### gx becomes a beloved tool by the Go Core Contributors Travis are you still working on this?
+* go-ipfs doesn't have checked-in gx paths
```

Comment:
@travisperson is this still a goal of yours? I don't know if you're still working on gx udpates or not.

Reply:
Yah - absolutely agree; one of those is going to be an OKR. And possibly even both.

Reply:
"Optimizing developer velocity" could be the objective. The KR would be "go-ipfs developers are no longer adversly affected by gx" (with some significant rewording). That way, either getting a maintainer or moving away from gx are valid solutions.